index.html

```diff
@@ -10,23 +10,246 @@
 
 <body class="body">
 
-  <div class="navbar">
-    <div class="my-logo">My Logo</div>
-    <a class="menu-icon"></a>
-  </div>
-
-  <div class="main">
-    <div class="primary-text">I'm Juan Diaz Glad to see you!</div>
-    <div class="secondary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
-    <div class="connect-text">LET'S CONNECT</div>
-    <div class="socialmedia-icons">
-      <a class="twitter-icon"></a>
-      <a class="linkedin-icon"></a>
-      <a class="m-icon"></a>
-      <a class="cat-icon"></a>
-      <a class="fingers-icon"></a>
-    </div>
-  </div>
+  <section class="intro">
+    <header>
+      <nav class="navbar">
+        <a class="my-logo" href="#">My Logo</a>
+        <div class="navlist">
+        <a class="navlink" href="#portfolio">Portfolio</a>
+        <a class="navlink" href="#about">About</a>
+        <a class="navlink" href="#contactme">Contact</a>
+        <a class="menu-icon" href="#"></a>
+        </div>
+      </nav>
+    </header>
```

Comment:
- [ ] [OPTIONAL] Great Job with your desktop version :+1: , however, I think you could work on your navigation bar and make it stick as much as possible to the Figma template, it is very important as a full-stack developer to peer attention to details. please make sure to have the same colors, font, and background :smile:

### Figma template
![Screenshot from 2022-07-07 09-59-46](https://user-images.githubusercontent.com/6615394/177747558-17363908-a602-4657-83a5-444f10c5b1c5.png)

### Your project
![Screenshot from 2022-07-07 09-52-47](https://user-images.githubusercontent.com/6615394/177747565-3c061429-26d0-4eff-8ff0-b6e84ccccf85.png)